samtranslator/policy_templates_data/policy_templates.json

```diff
@@ -1930,6 +1930,25 @@
           }
         ]
       }
-    }
+    },
+    "ApiGatewayFullPolicy": {
```

Comment:
There is a typo in the access policy name: `ApiGatewayFullAccessPolicy`

Reply:
I agree. ApiGatewayFullAccessPolicy is more appropriate.